src/services/msal.ts

```
@@ -1,22 +1,28 @@
-            logLevel: LogLevel.Verbose,
-        }
+            system: {
+                loggerOptions: {
+                    loggerCallback(_loglevel, message, _containsPii) {
+                        console.log(message);
+                    },
+                    piiLoggingEnabled: false,
+                    logLevel: LogLevel.Verbose,
+                }
+            }
+        });
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)